README.md

```diff
@@ -64,6 +66,12 @@ None.
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
 | geoip2\_database\_directory | The directory in which to store the database files. | `/usr/local/share/GeoIP/` | No |
+| geoip2\_geoipupdate\_auto\_update | Whether to configure automatic updates when `geoipupdate` is installed. | `true` | No |
+| geoip2\_geoipupdate\_service\_name | The name to use for the `geoipupdate` SystemD service and timer. | `geoipupdate` | No |
+| geoip2\_geoipupdate\_service\_timer\_on\_calendar | The calendar expression to use for the `geoipupdate` SystemD timer. | `Wed,Sat America/New_York` | No |
+| geoip2\_geoipupdate\_service\_timer\_randomized\_delay\_sec | The randomized delay in seconds to use for the `geoipupdate` SystemD timer. | `3h` | No |
```

Comment:
I'm confused about the description here talking about the value being in "seconds", but the default value is `3h`.  Shouldn't that be `10800` seconds?

Reply:
Please let me know if [9c1a1eb](https://github.com/cisagov/ansible-role-geoip2/pull/51/commits/9c1a1eb0d15c22bfd6ee139b38d48dc7d715efdf) clarifies things.

Reply:
Please see [450b394](https://github.com/cisagov/ansible-role-geoip2/pull/51/commits/450b394807a674789207cfc2951c2d16a59c19e1). I do not want to burn time trying to document the peculiarities of systemd's functionality, so I would rather just reference the options being populated and leave it to the reader.

Reply:
This works for me- I'm not dying on this hill.